_syllabus/seed.py

```diff
@@ -76,12 +77,19 @@
         hwDate  = lecDate.replace(hours=15)
         solDate = hwDate.shift(weeks=1)
 
+
+	 
+	oh_plus_plus_link = wk{0:02d}-ohpp.md'.format(idx + 1)
+
```

Comment:
To be clear: _I really appreciate your taking the time to investigate the repo, find and understand this file, and make a rational change._ I just wanted to double check with someone on staff, since I'm not sure that my concern is warranted. In fact, I worry that I am giving the issue more attention than is necessary with each passing character. = P